solr/core/src/java/org/apache/solr/metrics/SolrCoreMetricManager.java

```diff
@@ -147,7 +143,11 @@ public void registerMetricProducer(String scope, SolrMetricProducer producer) {
           "scope = " + scope + ", producer = " + producer);
     }
     // use deprecated method for back-compat, remove in 9.0
-    producer.initializeMetrics(solrMetricsContext.metricManager, solrMetricsContext.registry, solrMetricsContext.tag, scope);
+    if (!coreLevelMetricsEnabled && producer instanceof RequestHandlerBase) {
```

Comment:
Just to confirm my understanding - so this is done such that `coreLevelMetricsEnabled=false` is only effective on Request handler related metrics. Ie other metrics will always be aggregated under core?

Reply:
Correct, I can add a comment to make that more clear. This would solve the issue we saw with splits last night because it'll only aggregate things like request count/times at the node level for request handlers, but will preserve things like index size at the core level which overseer was using.